CHANGELOG.md

```diff
@@ -140,7 +145,8 @@ Initial release on [PyPi](https://pypi.org/)
 
 Initial release on [crates.io](https://crates.io/)
 
-[unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.0...HEAD
+[unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.1...HEAD
```

Comment:
The latest tag is `rust-0.4.1`, not `python-0.4.1`...the python tag doesn't exist

Reply:
🤦